Modules/CalcOffence-3_0.lua

```diff
@@ -1555,10 +1555,11 @@ function calcs.offence(env, actor, activeSkill)
 				end
 				if env.mode_effective then
 					local enemyInc = 1 + enemyDB:Sum("INC", nil, "SelfCritMultiplier") / 100
+					extraDamage = extraDamage + enemyDB:Sum("BASE", nil, "SelfCritMultiplier") / 100
 					extraDamage = round(extraDamage * enemyInc, 2)
 					if breakdown and enemyInc ~= 1 then
 						breakdown.CritMultiplier = {
-							s_format("%d%% ^8(additional extra damage)", skillModList:Sum("BASE", cfg, "CritMultiplier") / 100),
+							s_format("%d%% ^8(additional extra damage)", (enemyDB:Sum("BASE", nil, "SelfCritMultiplier")+skillModList:Sum("BASE", cfg, "CritMultiplier")) / 100),
```

Comment:
Can you add spaces around the '+' in (enemyDB:Sum("BASE", nil, "SelfCritMultiplier")+skillModList:Sum("BASE", cfg, "CritMultiplier") to maintain consistency please?

Reply:
fixed